selectors_solana.yml

```diff
@@ -0,0 +1,11 @@
+selectors:
+  "5eykt4UsFv8P8NJdTREpY1vzqKqZKvdpKuc147dw2N9d":
```

Comment:
What kind of string is this? Are there any docs we can link to in solana.go?

Reply:
its a base58 encoded 32 byte hash. +1 we should add a link to where it came from and some comments explaining that